README.md

```diff
@@ -1 +1,46 @@
-# FashionApp-team3
\ No newline at end of file
+# FashionApp-team3 
+
+## You can check the app [here]().
+
+
+## Team Members:-
+
+- [Aref AlRayes](https://github.com/arefalrayes)
+- [Saif Al Hayek](https://github.com/SaifHayek)
+- [Shatha K. Eqdaih](https://github.com/shathakh)
+- [Alaa Eid](https://github.com/AlaaEid-1) 
+
+
+## Why we created this app?
+
+- To help women to buy clothes easily in the midst of the covid-19 pandemic from their home. 
+- The sheer choice in shapes, sizes, color combinations, compositions, and budget.
+- It allows you to have a stress-free shopping experience.
+- You can access a lot of discounts.
+- It saves time and effort.
+
+## What is Fashion app?
+
+An Online premium clothing store for women which contains all their needs such as T-shirts, make-up, scarfs, jackets or coats. It can be accessed from both smartphones and laptops.  
```

Comment:
Put the (what) section before the (why) section.

Reply:
Ok